components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/OldTokensCleanDAO.java

```diff
@@ -168,6 +177,27 @@ private void removeTokenFromMainTable(String oldAccessTokenID, Connection connec
         }
     }
 
+    private void removeTokenFromMainTableByTokenHash(String accessTokenHash, Connection connection)
+            throws SQLException {
+
+        connection.setAutoCommit(false);
+        try {
+            PreparedStatement deletefromaccesstokentable =
```

Comment:
can we rename the variable to `deleteFromAccessTokenTable`?

Reply:
resolved with 80f91c3e4c0e801dd59f2b6711f6a351e4be327c